torch/testing/_internal/common_methods_invocations.py

```diff
@@ -12961,6 +12974,40 @@ def reference_flatten(input, start_dim=0, end_dim=-1):
                             "TestCompositeCompliance", "test_forward_ad"),
            )
            ),
+    OpInfo('batch_norm_with_update',
+           op=torch.ops.aten.batch_norm_with_update,
+           aten_name='batch_norm_with_update',
+           dtypes=floating_types_and(torch.float16, torch.bfloat16),
+           supports_forward_ad=True,
+           supports_fwgrad_bwgrad=True,
+           assert_jit_shape_analysis=True,
+           sample_inputs_func=sample_inputs_batch_norm_with_update,
+           skips=(
```

Comment:
What are all these skips? Inherited from the regular batchnomr?

Reply:
Yes, most of them are inherited. The new ones specific to `batch_norm_with_update` are due to either (1) the lack of python bindings (which also causes some out variant tests to fail), and (2) the fact that we decided for the new op to not handle contiguous inputs for cuda.

For (2), @bdhirsh and I discussed this in great detail. The problem was the cudnn and miopen ops can only handle contiguous args, and previously we would always force the args to be contiguous before calling the ops ([code](https://github.com/pytorch/pytorch/blob/main/aten/src/ATen/native/Normalization.cpp#L547)). If we just try to do this inside `batch_norm_with_update`, however, then the backward pass won't get the same contiguous inputs from the forward pass and will error. Therefore, we need to force the args to be contiguous before calling `batch_norm_with_update`, which is why this PR moves the contiguous calls out of the op.